Gem/Root/Navigation/Wallets/WalletsNavigationStack.swift

```diff
@@ -50,13 +49,13 @@ struct WalletsNavigationStack: View {
             }
             .sheet(isPresented: $isPresentingCreateWalletSheet) {
                 CreateWalletNavigationStack(
-                    keystore: keystore,
+                    manageWalletService: manageWalletService,
```

Comment:
```suggestion
                    manageWalletService: manageWalletService,
```

is it possible to rename `manageWalletService` to just `WalletService` to keep it simpler?